calicoctl/commands/get.go

```diff
@@ -47,18 +47,13 @@ Options:
   -o --output=<OUTPUT FORMAT>  Output format.  One of: yaml, json, ps, wide,
```

Comment:
Need to include `yml` in this list now.

Reply:
It's not a different option, it's just if someone types `yml` instead of `yaml` since people use them both interchangeably. This is like how we accept `ipp` for `ippool`, we don't list all the short versions we accept, just the right one

Reply:
yeah i'm not fussed about this one.